src/components/property-items/string-item/index.tsx

```diff
@@ -53,18 +60,21 @@ const StyledInput = styled.input`
 `;
 
 export const StringItem: React.StatelessComponent<StringItemProps> = props => {
-	const { className, onChange, onBlur, label, value } = props;
+	const { className, description, onChange, onBlur, label, value } = props;
 
 	return (
 		<StyledStringItem className={className}>
-			<PropertyLabel label={label} />
-			<StyledInput
-				onChange={onChange}
-				onBlur={onBlur}
-				type="text"
-				value={value || ''}
-				placeholder="…"
-			/>
+			<StyledContainer>
+				<PropertyLabel label={label} />
+				<StyledInput
+					onChange={onChange}
+					onBlur={onBlur}
+					type="text"
+					value={value || ''}
+					placeholder="…"
+				/>
+			</StyledContainer>
+			{description !== '' && <PropertyDescription description={description || ''} />}
```

Comment:
The description is still shown as empty element – but I only want to show it when it is not ''. Can you help me with that @marionebl?